README.md

```diff
@@ -1,4 +1,4 @@
-# TODO Library 
+# TODO Library
```

Comment:
Thank you a lot! I like the MR. 

Could you run `cargo fmt` and update the MR after that?

Reply:
Thanks for your review too!